manifests/init.pp

```diff
@@ -242,7 +258,7 @@
       }
       contain foreman_proxy_content::dispatch_router
 
-      if $pulp_master {
+      unless $enable_pulp2_mirror {
```

Comment:
Can you use `unless` if there is also an `else`? Maybe it was Ruby where that's not possible.